### PR TITLE
Make Spark2 example more idiomatic

### PR DESCRIPTION
**Pull request by donderom**
The idea is to use SparkSession for Spark integration example as it's more idiomatic way in Spark 2.

### CODE REVIEW

File: integration/src/test/resources/ammonite/integration/basic/Spark2.json [hunk 1]
```diff
@@ -1,6 +1,4 @@
-[
-  {"hello": 1},
-  {"hello": 2},
+{"hello": 1},
+{"hello": 2},
 
-  {"hello": 3}
-]
+{"hello": 3}
```
Comment (lihaoyi) [inline integration/src/test/resources/ammonite/integration/basic/Spark2.json]: Uhhh... this isn't a JSON file anymore? Is spark fine with that?

**Comment by lihaoyi** [comment]
Looks good i guess, except for the weird non-JSON file... not saying it's wrong, but if you could tell me why it isn't wrong I'll merge this

File: integration/src/test/resources/ammonite/integration/basic/Spark2.json (hunk 1 above)
Comment (donderom) [inline integration/src/test/resources/ammonite/integration/basic/Spark2.json]: That's how Spark JSON reader works by default. With the current Spark2.json the result of `df` would be:
```
+---------------+-----+
|_corrupt_record|hello|
+---------------+-----+
|              [| null|
|           null|    1|
|           null|    2|
|           null|    3|
|              ]| null|
+---------------+-----+
```

Now it produces the following:
```
+-----+
|hello|
+-----+
|    1|
|    2|
|    3|
+-----+
```
The official Spark documentation uses the same format for their examples.

Comment (lihaoyi) [inline integration/src/test/resources/ammonite/integration/basic/Spark2.json]: Huh ok, well I guess this is great then

**Pull request merged**